src/status/mod.rs

```diff
@@ -18,6 +20,20 @@ pub enum ChatterLevel {
     Normal,
 }
 
+impl FromStr for ChatterLevel {
+    type Err = &'static str;
+
+    fn from_str(a_str: &str) -> StdResult<Self, Self::Err> {
+        let actual = match a_str {
+            "default" => ChatterLevel::Normal,
+            "minimal" => ChatterLevel::Minimal,
+            _ => unreachable!(),
```

Comment:
This `FromStr` impl still needs changing away from `unreachable!`.

Reply:
Ah, thanks for catching that.